libs/blocks/featured-article/featured-article.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/named, import/extensions */
+
+import {
+  buildArticleCard,
+  getBlogArticle,
+  loadTaxonomy,
+} from '../article-feed/article-helpers.js';
+
+async function decorateFeaturedArticle(featuredArticleEl, articlePath, eager = false) {
+  const article = await getBlogArticle(articlePath);
+  if (article) {
+    const card = buildArticleCard(article, 'featured-article', eager);
+    const tagHeader = document.querySelector('.tag-header-container > div');
+    if (tagHeader) {
+      featuredArticleEl.append(card);
+      tagHeader.append(featuredArticleEl);
+    } else {
+      featuredArticleEl.append(card);
+    }
+  } else {
+    const { origin } = new URL(window.location.href);
+    // eslint-disable-next-line no-console
+    console.warn(`Featured article does not exist or is missing in index: ${origin}${articlePath}`);
```

Comment:
you're already kind of logging the same thing in your getMetadataJson method if a request fails. 
Additionaly this could be rewritten to just 
```
console.warn(`Featured article does not exist or is missing in index: ${window.location.origin}${articlePath}`);
```
but personally I'd just remove